packages/java/tests/spring/endpoints-latest-java/pom.xml

```diff
@@ -12,9 +12,9 @@
     <packaging>jar</packaging>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
-        <maven.compiler.release>17</maven.compiler.release>
+        <maven.compiler.source>20</maven.compiler.source>
```

Comment:
I would jump to 21 directly

Reply:
Well it seems that GH Actions doesn't support installing the 21 yet.

Reply:
True, it will probably not appear until next week

Reply:
Is 21 LTS? I would prefer to stick with LTS version to not ship code that cannot compile with LTS